packages/kotti-table/src/Table.vue

```diff
@@ -46,9 +46,10 @@ export default {
 	components: { TableBody, TableHeader, TableColumn },
 	name: 'KtTable',
 	props: {
+		id: { default: null, type: String },
 		rowKey: { type: String },
-		rows: { required: true, type: Array },
-		columns: { default: () => [], type: Array },
+		rows: { default: () => [], type: Array },
+		columns: { type: Array },
```

Comment:
Why remove the `default` here, and further-down create an implicit default? Why is this not a `required` property if it doesn’t have a `default`?

Reply:
I recall I had a reason when I was fixing one of the bugs I'll reset it to default to null and see what happens